azure-pipelines.yml

```diff
@@ -1,5 +1,14 @@
 name: merq
 
+parameters:
```

Comment:
I ended up removing the Signing input parameter from `Clide` to avoid making it too convenient for someone (aka bad actor) to launch a real signed build against any branch. Recommend you do the same

Reply:
Resolved in [a1638c2](https://github.com/xamarin/Merq/pull/4/commits/a1638c2700063c6351a65c4d452b18c47d4b7b26)